content/en/docs/concepts/cluster-administration/swap-memory-management.md

```diff
@@ -0,0 +1,397 @@
+  to not be supported, hence will not be used.
+  A kubelet log entry will be emitted to warn the user about memory-backed volumes might swap to disk.
+  If kubelet succeeds, the dummy tmpfs will be deleted and the `noswap` option will be used.
+  * If the `noswap` option is not supported, kubelet will emit a warning log entry,
+    then continue its execution.
+
+See the [section above](#setting-up-encrypted-swap) with an example for setting unencrypted swap.
+However, handling encrypted swap is not within the scope of kubelet;
+rather, it is a general OS configuration concern and should be addressed at that level.
+It is the administrator's responsibility to provision encrypted swap to mitigate this risk.
+
+### Evictions
+
+Configuring memory eviction thresholds for swap-enabled nodes can be tricky.
+
+With swap being disabled, it is reasonable to configure kubelet's eviction thresholds
+to be a bit lower than the node's memory capacity.
+The rationale is that we want Kubernetes to start evicting Pods before the node runs out of memory
+and invokes the Out Of Memory (OOM) killer, since the OOM killer is not Kubernetes-aware,
+therefore does not consider things like QoS, pod priority, or other Kubernetes-specific factors.
+
+With swap enabled, the situation is more complex.
+In Linux, the `vm.min_free_kbytes` parameter the memory threshold for the kernel
+to start aggressively reclaiming memory, which includes swapping out pages.
+If the kubelet's eviction thresholds are set in a way that eviction would take place
+before the kernel starts reclaiming memory, it could lead to workloads never
+being able to swap out during node memory pressure.
+However, setting the eviction thresholds too high could result in the node running out of memory
+and invoking the OOM killer, which is not ideal either.
+
+To address this, it is recommended to set the kubelet's eviction thresholds
+to be slightly lower than the `vm.min_free_kbytes` value.
+This way, the node can start swapping before kubelet would start evicting Pods,
+allowing workloads to swap out unused data and preventing evictions from happening.
+On the other hand, since it is just slightly lower, kubelet is likely to start evicting Pods
+before the node runs out of memory, thus avoiding the OOM killer.
+
+The value of `vm.min_free_kbytes` can be determined by running the following command on the node:
+```shell
+cat /proc/sys/vm/min_free_kbytes
+```
+
+### Unutilized swap space
+
+Under the `LimitedSwap` behavior, the amount of swap available to a Pod is determined automatically,
+based on the proportion of the memory requested relative to the node's total memory
+(For more details, see the [section below](#how-is-the-swap-limit-being-determined-with-limitedswap)).
+
+This design means that usually there would be some portion of swap that will remain
+restricted for Kubernetes workloads.
+For example, since Guaranteed QoS pods are currently not permitted to use swap,
```

Comment:
This is the first time you mention this limitation. I think you need to mention that LimitedSwap is only available for Burstable Pods.

Reply:
Thanks for catching that!

For context, this section used to be one of the firsts to appear, but due to [this feedback](https://github.com/kubernetes/website/pull/51303#discussion_r2162116246) I've moved it to the end.

Now, under `## Enabling swap for Kubernetes Workloads`, I've added a brief explanation of the excluded pods with a pointer to the more detailed section below:
> Bear in mind that the following pods would be excluded from swap access (see more info in the [section below](#how-is-the-swap-limit-being-determined-with-limitedswap)):
> - Pods that are not classified as Burstable QoS.
> - Pods of High-priority.
> - Containers with memory limit that equals to memory request.

WDYT?